### PR TITLE
Fix transactions pagination on root page

### PR DESCRIPTION
**Pull request by Gataniel**

### CODE REVIEW

File: spec/features/transactions_list_spec.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
+
+describe 'Transactions list' do
+  require "money-rails/test_helpers"
```
Comment (shir) [inline spec/features/transactions_list_spec.rb]: @Gataniel put require at the beginning of the file.

File: spec/features/transactions_list_spec.rb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    it "doesn't display another organization transactions" do
+      expect(subject).to_not have_content(humanized_money_with_symbol(org1_transaction.amount))
+    end
+  end
+
+  context "pagination" do
+    before do
+      15.times do
+        FactoryGirl.create(:transaction, bank_account: org1_ba, amount: rand(1..9999.99))
```
Comment (shir) [inline spec/features/transactions_list_spec.rb]: @Gataniel instead `15.times` use `create_list :transaction, 15, ...`

File: spec/features/transactions_list_spec.rb [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      end
+      visit root_path
+    end
+
+    it "lists 10 first transactions" do
+      within ".transactions" do
+        org1_ba.transactions.first(10).each do |transaction|
```
Comment (shir) [inline spec/features/transactions_list_spec.rb]: @Gataniel instead checking for organization transactions check if transactions which you've create with factory are exists.

Comment (Gataniel) [inline spec/features/transactions_list_spec.rb]: @shir didn't get it, could you pls explain. As for me existence doesn't imply that pagination works

File: spec/features/transactions_list_spec.rb [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+    it "doesn't display another organization transactions" do
+      expect(subject).to_not have_content(humanized_money_with_symbol(org1_transaction.amount))
+    end
+  end
+
+  context "pagination" do
+    let(:transactions) { [] }
+    before do
+      transactions = FactoryGirl.create_list :transaction, 15, bank_account: org1_ba
```
Comment (shir) [inline spec/features/transactions_list_spec.rb]: @Gataniel why don't you put in `let`? If you want to create it before visit then use `let!`

**Pull request merged**